mds/models.py

```diff
@@ -97,6 +97,7 @@ class Provider(models.Model):
     id = models.UUIDField(primary_key=True, default=uuid.uuid4)
     name = UnboundedCharField(blank=True, default="")
     logo_b64 = UnboundedCharField(null=True, blank=True, default=None)
+    operator = models.BooleanField(null=True)
```

Comment:
It can't stay `null`, `BooleanField` will only accept `True` and `False` values, so creating providers will fail.

This can only be a transient situation to migrate existing providers when we deploy.

Now you need two more migrations:
- one to fill in the new field for existing providers;
- one to restore the default value for newly created providers.

So you'll also have to choose a default value, i.e. whether newly created providers (from the frontend or the poller automatically creating them) must automatically show up in the filters.

Which makes me think this change will break upserting providers from the poller: https://github.com/Polyconseil/django-mds/blob/master/mds/db_helpers.py#L30-L52